addOns/requester/CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+## Added 
+- Added Button to lowercase Header Names
```

Comment:
> Button to lowercase request header names (Issue 8176).

Reply:
Completed, and I truly appreciate all your assistance. Thank you!